### ISSUE TITLE
erőmű limitálás

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#21]
```
Game version:0.92


Ectoplasma, és kráteres felszíneken nem építhető fúziós erőmű.
```

Original issue reported on code.google.com by `Jozsef.T...@gmail.com` on 14 Aug 2011 at 11:07

**Comment by GoogleCodeExporter** [comment]
```
Ez így van. Hogy sikerült mégis?
```

Original comment by `akarn...@gmail.com` on 15 Aug 2011 at 9:21

**Comment by GoogleCodeExporter** [comment]
```
Nem sikerült :) Eredeti játékban lehet építeni kráteres felszínre 
fúziós erőművet, és naperőművet is! Ektoplazma felszínre pedig nem csak 
atomerőművet, hanem naperőművet is lehet építeni. Latyakos típusra pedig 
atomerőművet, és fúziós erőművet is lehet építeni( ezt a típust még 
nem ellenőriztem) A többi limitáció rendben van.
```

Original comment by `Jozsef.T...@gmail.com` on 15 Aug 2011 at 11:39

**Comment by GoogleCodeExporter** [comment]
```
Utána néztem, valóban. Nem tudom, honnan szedtem a jelenlegit.

http://capsu.org/imperium/colonies_buildings.html

Atomerőmű: bárhol
Fúziós erőmű: kivéve sivatagos, neptoplazma, sziklás
Naperőmű: kivéve jeges, vizes




```

Original comment by `akarn...@gmail.com` on 15 Aug 2011 at 11:46

**Comment by GoogleCodeExporter** [comment]
```
Fix in 0.93.100
```

Original comment by `akarn...@gmail.com` on 17 Aug 2011 at 11:05
- Changed state: **Fixed**
